tests/internal/scripts/ccuda/kernel_launch.py

```diff
@@ -10,7 +10,7 @@ def func(a):
 if __name__ == '__main__':
     threads_per_block = 5
     n_blocks = 1
-    arr = cuda.array([0, 1, 2, 3, 4])
+    arr = cuda.array([0, 1, 2, 3, 4], memory_location='host')
```

Comment:
this array should be either `managed` or `device` as we are using it as a `kernel` argument